Makefile.am

```diff
@@ -594,7 +594,7 @@ netdata_SOURCES = $(NETDATA_FILES)
 
 if ENABLE_ACLK
 netdata_LDADD = \
-    mosquitto/lib/libmosquitto.a \
+    externaldeps/mosquitto/lib/libmosquitto.a \
```

Comment:
This doesn't match the installer (@Ferroin please confirm). It should be `externaldeps/mosquitto/libmosquitto.a`.